cmd/geth/usage.go

```diff
@@ -249,6 +249,7 @@ var AppHelpFlagGroups = []flagGroup{
 			utils.ProxiedValidatorAddressFlag,
 			utils.ProxiedFlag,
 			utils.ProxyEnodeURLPairFlag,
+			utils.ProxyOverrideAnnounceIPCheckFlag,
```

Comment:
naming nit: how about `ProxyAllowExternalPrivateIPFlag` ? we don't call it the announce ip elsewhere do we?